selfdrive/controls/tests/test_startup.py

```diff
@@ -49,6 +56,9 @@ class TestStartup(unittest.TestCase):
     (EventName.startupMaster, HYUNDAI.KIA_STINGER, True, None),
     (EventName.communityFeatureDisallowed, HYUNDAI.KIA_STINGER, False, None),
 
+    # dashcamOnly, FW query
+    (EventName.startupNoControl, MAZDA.CX5, False, CX5_FW_VERSIONS),
+
     # dashcamOnly car
     (EventName.startupNoControl, MAZDA.CX5, True, None),
     (EventName.startupNoControl, MAZDA.CX5, False, None),
```

Comment:
```suggestion
    (EventName.startupNoControl, MAZDA.CX5, True, CX5_FW_VERSIONS),
    (EventName.startupNoControl, MAZDA.CX5, False, CX5_FW_VERSIONS),
```

Reply:
I'm aware @jyoung8607 , was trying to get the new thing to work first. Thanks for your help addressing the failed tests.